packages/xstate-wallet/src/workflows/virtual-defunding-as-leaf.ts

```diff
@@ -108,15 +126,47 @@ export const defundGuarantorInLedger = (store: Store) => async ({
   ]);
   return {state: nextState(ledgerState, outcome)};
 };
+export {defundGuarantorInLedger};
+
+const acquireLock = (store: Store) => async ({ledgerId}: ChannelsSet): Promise<ChannelLock> => {
```

Comment:
This is copied from ledger-funding, and simplified in https://github.com/statechannels/monorepo/pull/1809/commits/2a557630c9f8a759402329614fc61ff94db01e58